blib/Perl6/BOOTSTRAP/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Don't remove this file and dir.
+# They are needed to host the bytecode files during the build process
+for any VM.
```

Comment:
Missing `#` at the start of this line?

Reply:
Actually, same we have in `blib/Perl6`. I copied it over without actually checking its content.